pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/api/v2/WorkerApiV2Resource.java

```diff
@@ -132,4 +140,21 @@ public WorkerInfo getClusterLeader() {
     public List<ConnectorDefinition> getConnectorsList() throws IOException {
         return worker.getListOfConnectors(clientAppId());
     }
+
+    @GET
+    @ApiOperation(
+            value = "Determines whether the worker service is initialized and ready for use",
+            response = Boolean.class
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Invalid request"),
+            @ApiResponse(code = 408, message = "Request timeout")
+    })
+    @Path("/initialized")
```

Comment:
What is the full path for this? Will callers need to be authenticated? If so we might want to make that optional (like the metrics endpoint). Otherwise, using this as a readiness/liveness probe in kubernetes will be difficult.